icssc-projects/peterportal-client
# Ayush

Added functionality to undo votes on reviews, colored vote buttons if user has already voted on the review

## Description
Added a new patch to the review router to get the color for a specific vote for a review and user, used the useEffect React hook to constantly update the className for each vote button.

## Screenshots

Before:
![image](https://user-images.githubusercontent.com/18633636/157330153-78888686-26c1-4cff-9283-e3d118496750.png)
After:
![image](https://user-images.githubusercontent.com/18633636/157330256-e1e404bb-2634-4013-8cdc-aac7ef999b25.png)


## Steps to verify/test this change:
Go to any course(course used to test COMPSCI 143A) and vote on any of the test reviews there, see how the buttons color when you vote and click again to remove the vote.
